src/components/cartMobile.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useRef } from 'react';
+
+interface CartProps {
+  isShown: boolean;
+  closeCart: () => void;
+}
+
+const Cart: React.FC<CartProps> = ({ isShown, closeCart }) => {
```

Comment:
React.FC sert à rien et c'est une mauvaise pratique

```suggestion
const Cart = ({ isShown, closeCart }: CartProps) => {
```

Reply:
Ce composant n'est pas utile, à la place il faudrait copier et déplacer le node du drawer déjà présent dans la page pour l'ajouter dans le bouton

En dev, tu peux peut-être l'ajouter au mock.html